Popularity/DCAFPilot/setup.sh

```diff
@@ -5,3 +5,5 @@ export PATH=$PWD/bin:$PATH
 if [ -f /data/srv/current/apps/das/etc/profile.d/init.sh ]; then
 source /data/srv/current/apps/das/etc/profile.d/init.sh
 fi
+
+source /data/srv/current/apps/DCAFPilot/etc/profile.d/init.sh
```

Comment:
Wrap it in if then block as previous block does. This will ensure that bash script will not fail if init.sh does not exists on a system